### ISSUE TITLE
Fails with Nextflow version  24.04.2

### ISSUE THREAD

**Issue by chriswyatt1** [Eco-Flow/excon#7]
Same for all eco flow pipelines, it is failing with newer versions of nextflow with the following error:\

```
ERROR ~ Error executing process > 'DOWNLOAD_NCBI (Bombus_terrestris via GCF_910591885.1)'

Caused by:
  No signature of method: nextflow.executor.BashFunLib.render() is applicable for argument types: () values: []
  Possible solutions: render(), every(), find(), every(groovy.lang.Closure), find(groovy.lang.Closure), any()

 -- Check '.nextflow.log' file for details
```

**Comment by chriswyatt1** [comment]
In mean time, you can run with :

`NXF_VER=23.10.1 nextflow run main.nf -resume -profile docker,test_small`

**Comment by chriswyatt1** [comment]
fixed with [c17e379](https://github.com/Eco-Flow/excon/commit/c17e379cb8d71f1c1397bb052c4a6cf8d784bc34)
